### PR TITLE
Adds note about origin in writing a plan section of docs

### LINKED ISSUE #2578: [Documentation] How to use third party origins
Mention here, how to use third party Origins like Xorg: https://www.habitat.sh/docs/create-plans/

### PR DESCRIPTION
**Pull request by rsertelon**
Closes #2578

Signed-off-by: Romain Sertelon <romain@sertelon.fr>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by ShalokShalom** [comment]
Yes, fine.

File: www/source/partials/docs/_dev-pkgs-write-plans.html.md.erb [hunk 1]
```diff
@@ -27,6 +27,8 @@ pkg_bin_dirs=(bin)
 
 It has the name of the software, the version, where to download it, a checksum to verify the contents are what we expect, run dependencies on `core/glibc` and `core/readline`, build dependencies on `core/coreutils`, `core/make`, `core/gcc`, libraries files in `lib`, header files in `include`, and a binary file in `bin`. Also, because it's a core plan, it has a description and upstream URL for the source project included.
 
+> Note: the `core` prefix is the origin of those dependencies. For more information, see [Create an Origin](/tutorials/download/create-origin/)
```
Comment (davidwrede) [inline www/source/partials/docs/_dev-pkgs-write-plans.html.md.erb]: Looks like there's a typo in the beginning of this note. "Note: The `core`...."

Comment (rsertelon) [reply www/source/partials/docs/_dev-pkgs-write-plans.html.md.erb]: @davidwrede fixed :)

**Pull request merged**